docs-website/package.json

```diff
@@ -49,5 +37,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "engines": {
+    "node": ">=16.14"
```

Comment:
If we are using Node 18 in CI, should we specify Node 18+ here as well?

Reply:
removed